Baekjoon/11650.py

```diff
@@ -0,0 +1,31 @@
+'''
+문제 링크:https://www.acmicpc.net/problem/11650
+카테고리: 정렬
+문제해설: 좌표 정렬하기
+2차원 평면에 점 n개를 좌표 순으로 정렬하기
+1. x좌표 증가하는 순서
+2. x좌표 같으면 y좌표 증가하는 순서
+
+for문 2개->시간 초과
+퀵소트 or 머지소트->시간 복잡도 최악이어도(nlogn) or 내장 함수
+'''
+import sys
+
+input = sys.stdin.readline
+
+n = int(input())
+
+cor = []
+
+for i in range(n):
+  x, y = map(int, input().split())
+  cor.append([x, y])
```

Comment:
```suggestion
  x = list(map(int, input().split()))
  cor.append(x)
```
바로 리스트로 받아서 append 해도 될 것 같네요